backend/fms_core/services/experiment_run_info.py

```diff
@@ -155,11 +155,7 @@ def _generate_run_info_samples(experiment_run: ExperimentRun) -> List[RunInfoSam
     samples = Sample.objects.filter(container=experiment_run.container)
 
     for sample in samples:
-        if (sample.is_pool):
-            generated_rows += _generate_pooled_samples(experiment_run, sample)
-        else:
-            run_info_sample = _generate_sample(experiment_run, sample, sample.derived_sample_not_pool)
-            generated_rows += [run_info_sample]
+        generated_rows += _generate_pooled_samples(experiment_run, sample)
```

Comment:
You sure you don't need to check if the sample is actually a pool?

Reply:
All samples are pools of one ...